docs/user_guide/assessment.rst

```diff
@@ -258,6 +258,223 @@ across groups and also the difference and ratio between the maximum and minimum:
+
+In the case of regression, a predictor :math:`f` satisfies demographic parity
+under a distribution over :math:`(X, A, Y)` if :math:`f(X)` is independent
+of the sensitive feature :math:`A`. This is equivalent to
+:math:`\P[f(X) \geq z \given A=a] = \P[f(X) \geq z] \quad \forall a, z`.
+:footcite:`agarwal2019fair` Another way to think of demographic parity in a 
+regression scenario is to compare the average predicted value across groups.
+Note that in the Fairlearn API, :func:`fairlearn.metrics.demographic_parity_difference` 
+is only defined for classification. 
+
+.. note::
+   Demographic parity is also sometimes referred to as *independence*, *group fairness*, *statistical parity*, and *disparate impact*.
+
+Failing to achieve demographic parity could generate allocation harms. 
+Allocation harms occur when AI systems allocate 
+opportunities, resources, or information differently across different 
+groups (for example, an AI hiring system that is more likely to advance resumes 
+of male applicants than resumes of female applicants regardless of qualification). 
+Demographic parity can be used to assess the extent of allocation harms because it 
+reflects an assumption that resources should be allocated proportionally 
+across groups. Of the metrics described in this section, it can be the easiest 
+to implement. However, operationalizing fairness using demographic parity 
+rests on a few assumptions: that either the dataset is not a good representation  
+of what the world actually looks like (e.g., a resume assessment system that is 
+more likely to filter out qualified female applicants due to an organizational 
+bias towards male applicants, regardless of skill level), or that the dataset 
+is an accurate representation of the phenomena being modeled, but the 
+phenomena itself is unjust. In reality, this may not be the case. The
+dataset might be an accurate representation of the phenomena itself, 
+or the phenomena being modeled may not be unjust. 
+If either assumption is not true, then demographic parity may not provide 
+a meaningful or useful measurement of the fairness of a model's predictions. 
+
+Fairness metrics like demographic parity can also be used as optimization 
+constraints during the machine learning model training process. However, 
+demographic parity may not be well-suited for this purpose because 
+it does not place requirements on the exact distribution of predictions with 
+respect to other important variables. To understand this concept further, 
+consider an example from the Fairness in Machine Learning textbook :footcite:`fairml_book`. 
+
+    "However, decisions based on a classifier that satisfies independence can 
+    have undesirable properties (and similar arguments apply to other 
+    statistical critiera). Here is one way in which this can happen, 
+    which is easiest to illustrate if we imagine a callous or ill-intentioned 
+    decision maker. Imagine a company that in group a hires diligently 
```

Comment:
Could we italicize or capitalize the "group a" and "group b" so it's a little bit easier to read?

Reply:
Done!